### ISSUE TITLE
Is JDK 8 is minimum requirement?

### ISSUE THREAD

**Issue by yurilaguta** [zsoltk/paperwork#38]
Hi there, 

I've decided to try this awesome library and was faced with:

Error: (3,0) Cause: hu/supercluster/paperwork/plugin/PaperworkPlugin : Unsupported major.minor version 52.0
Android Studio 2 Beta 2
OS X 10.11.3
java version "1.7.0_79"
Java(TM) SE Runtime Environment (build 1.7.0_79-b15)
Java HotSpot(TM) 64-Bit Server VM (build 24.79-b02, mixed mode)

**Comment by zsoltk** [comment]
Hello,

Thanks for bringing this to my attention! No, it shouldn't be required. Could you please add the runtime dependency from the snapshot repository, using the 1.2.4-SNAPSHOT version and confirm that it compiles?

Thanks!

**Comment by zsoltk** [comment]
To clarify:

``` groovy
buildscript {
    repositories {
        mavenCentral()
        maven { url "https://oss.sonatype.org/content/repositories/snapshots" }
    }

    dependencies {
        classpath 'com.android.tools.build:gradle:1.5.0'
        classpath 'hu.supercluster:paperwork-plugin:1.2.4-SNAPSHOT'
    }
}

apply plugin: 'com.android.application'
apply plugin: 'hu.supercluster.paperwork'

// ...

repositories {
    maven { url "https://oss.sonatype.org/content/repositories/snapshots" }
}

dependencies {
    compile 'hu.supercluster:paperwork:1.2.4-SNAPSHOT'
}
```

**Comment by yurilaguta** [comment]
Thanks. Works fine.

**Comment by zsoltk** [comment]
I released version 1.2.4. You can now revert to using the production repository with the new version instead of the snapshot one.

**Comment by yurilaguta** [comment]
Thanks

Best Regards,
Yura.
